<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>Graphics::Track</class>
    <widget class="QWidget" name="Graphics::Track">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>150</width>
                <height>120</height>
            </rect>
        </property>
        <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
            </sizepolicy>
        </property>
        <property name="minimumSize">
            <size>
                <width>150</width>
                <height>120</height>
            </size>
        </property>
        <property name="windowTitle">
            <string>Form</string>
        </property>
        <property name="styleSheet">
            <string notr="true"/>
        </property>
        <widget class="QWidget" name="verticalLayoutWidget">
            <property name="geometry">
                <rect>
                    <x>-1</x>
                    <y>-1</y>
                    <width>151</width>
                    <height>121</height>
                </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
                <property name="spacing">
                    <number>4</number>
                </property>
                <property name="leftMargin">
                    <number>0</number>
                </property>
                <property name="topMargin">
                    <number>0</number>
                </property>
                <property name="rightMargin">
                    <number>0</number>
                </property>
                <property name="bottomMargin">
                    <number>0</number>
                </property>
                <item>
                    <widget class="QLabel" name="trackName">
                        <property name="sizePolicy">
                            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                                <horstretch>0</horstretch>
                                <verstretch>0</verstretch>
                            </sizepolicy>
                        </property>
                        <property name="font">
                            <font>
                                <weight>75</weight>
                                <bold>true</bold>
                            </font>
                        </property>
                        <property name="text">
                            <string>Track's name</string>
                        </property>
                    </widget>
                </item>
                <item>
                    <spacer name="verticalSpacer">
                        <property name="orientation">
                            <enum>Qt::Vertical</enum>
                        </property>
                        <property name="sizeHint" stdset="0">
                            <size>
                                <width>20</width>
                                <height>40</height>
                            </size>
                        </property>
                    </spacer>
                </item>
                <item>
                    <layout class="QVBoxLayout" name="verticalLayout_2">
                        <property name="spacing">
                            <number>4</number>
                        </property>
                        <property name="leftMargin">
                            <number>5</number>
                        </property>
                        <property name="topMargin">
                            <number>5</number>
                        </property>
                        <property name="rightMargin">
                            <number>5</number>
                        </property>
                        <property name="bottomMargin">
                            <number>5</number>
                        </property>
                        <item>
                            <widget class="Graphics::Slider" name="volumeSlider">
                                <property name="sizePolicy">
                                    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                                        <horstretch>0</horstretch>
                                        <verstretch>0</verstretch>
                                    </sizepolicy>
                                </property>
                                <property name="value">
                                    <number>50</number>
                                </property>
                                <property name="orientation">
                                    <enum>Qt::Horizontal</enum>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="Graphics::Slider" name="panSlider">
                                <property name="sizePolicy">
                                    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                                        <horstretch>0</horstretch>
                                        <verstretch>0</verstretch>
                                    </sizepolicy>
                                </property>
                                <property name="orientation">
                                    <enum>Qt::Horizontal</enum>
                                </property>
                            </widget>
                        </item>
                    </layout>
                </item>
            </layout>
        </widget>
    </widget>
    <customwidgets>
        <customwidget>
            <class>Graphics::Slider</class>
            <extends>QSlider</extends>
            <header>Graphics/Widgets/slider.h</header>
        </customwidget>
    </customwidgets>
    <resources/>
    <connections/>
</ui>
